pom.xml

```diff
@@ -127,6 +127,7 @@
         <bitmap.version>1.0.1</bitmap.version>
         <prometheus.version>0.16.0</prometheus.version>
         <flink.version>1.18.1</flink.version>
+        <fabric8.version>6.11.0</fabric8.version>
```

Comment:
Is `kubernetes-client.version` a proper name here as the fabric8 does not only mean Kubernetes?

Reply:
Fabric8 kubernetes is the client an implementation, if use **kubernetes-client** name with fabric8. we may be confused with [kubernetes-client](https://github.com/kubernetes-client/java), use fabric8 client which has more powerful features

Reply:
Then how about `fabric8-kubernetes-client.version`?

Reply:
Now resolve to fabric8-kubernetes-client.version